### ISSUE TITLE
invalid Mid statement syntax, expected a call on the LHS

### ISSUE THREAD

**Issue by Kr00l** [WaynePhillipsEA/twinbasic#186]
**Describe the bug**
invalid Mid statement syntax, expected a call on the LHS

**To Reproduce**
```
Public Function GetDecimalChar() As String
GetDecimalChar = Mid$(CStr(1.1), 2, 1)
End Function
```

**Screenshots**
![image](https://user-images.githubusercontent.com/33756666/120775985-09de7300-c524-11eb-8b2f-608a6cc0b7bc.png)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]
 - twinBASIC compiler version [e.g. 0.10.933]

**Additional context**
n/a

**Comment by cristianbuse** [comment]
Probably related with the fix in #147.

If on the right hand side of the assignment, it needs to look at the ```Mid$``` function and ignore the ```Mid$``` statement.

**Comment by WaynePhillipsEA** [comment]
Apologies, this regression should have been picked up in our tests, but our test coverage of VBA.Strings.* functions is particularly lacking.  #187

**Comment by cristianbuse** [comment]
@WaynePhillipsEA  Would it be worth to open up the testing to the community via another repository? So that more is covered with less of your time.

**Comment by WaynePhillipsEA** [comment]
@cristianbuse yes, I think so.  I would like to see #54 implemented first, so we can start in a formal and structured manner.  I believe the necessary testing features in VS code are due to land in a few days.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.10.951, thanks for reporting
